code/modules/flufftext/Hallucination.dm

```diff
@@ -851,12 +995,8 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 	switch(sound_type)
 		if("phone")
 			target.playsound_local(source, 'sound/weapons/ring.ogg', 15)
-			sleep(25)
-			target.playsound_local(source, 'sound/weapons/ring.ogg', 15)
-			sleep(25)
-			target.playsound_local(source, 'sound/weapons/ring.ogg', 15)
-			sleep(25)
-			target.playsound_local(source, 'sound/weapons/ring.ogg', 15)
+			for (var/next_rings in 1 to 3)
+				addtimer(CALLBACK(target, /mob/.proc/playsound_local, source, 'sound/weapons/ring.ogg', 15), 25 * next_rings)
```

Comment:
If my understanding of the timer ss is correct these could all sound at the same time in case of lag, is that alright/correct?

Reply:
That's correct, I just didn't want to write out an entire process chain. 😛